chompfile.toml

```diff
@@ -43,6 +43,26 @@ target = 'dist/lexer.cjs'
 deps = ['dist/lexer.js']
 run = 'babel dist/lexer.js | terser -c -m -o dist/lexer.cjs'
 
+[[task]]
+name = 'build:swc'
+target = 'src/lexer.js'
+dep = 'src/lexer.ts'
+template = 'swc'
+[task.template-options]
+# TODO This still seems to generate lexer.js.map
+source-maps = false
```

Comment:
Seems this is an SWC issue - https://github.com/swc-project/cli/issues/113.

One easy fix might be to eject the SWC plugin usage (`chomp --eject`) can be used to see what this looks like, Ideally we could eject specific tasks but that isn't a feature currently.

Then the SWC task could be updated to:

```toml
run = '''
./node_modules/@swc/cli/bin/swc ....
rm src/lexer.ts.map
'''
```

Reply:
Ah ha, thanks for finding that! Applied the ejected output just for this task, with with a note that we should switch back to the swc template once that issue is fixed.